data_structures/k_tree/k_tree.py

```diff
@@ -79,3 +79,30 @@ def _walk(nodes):
 
         if self.root:
             _walk([self.root])
+
+    def print_level_order(self):
```

Comment:
According to the spec, this should be a function that takes a k-ary tree as an argument, not a method on the `KTree` class.